somnio_starter/__brick__/{{project_name.snakeCase()}}/pubspec.yaml

```diff
@@ -26,8 +26,8 @@ dependencies:
   go_router: ^14.2.9
   intl: ^0.19.0
   package_info_plus: ^8.0.2
-  {{short_name}}_ui:
-    path: packages/{{short_name}}_ui
+  {{project_name.snakeCase()}}_ui:
```

Comment:
We use short_name as a convention, right?

Reply:
I wasn't sure about using `short_name` or `project_name`, I used `project_name` since in the folders structure we are using it too. If we replace this with `project_name`, we should also change the ui package folder name to use it. Is that ok?